crates/ruma-common/src/events/message.rs

```diff
@@ -188,3 +189,12 @@ impl Text {
         "text/plain".to_owned()
     }
 }
+
+/// The error type returned when a conversion to an extensible event type fails.
+#[derive(Error, Debug)]
```

Comment:
I think we generally have builtin derives before thirdparty ones, can you swap them here?